packages/cli/generators/example/index.js

```diff
@@ -30,6 +30,9 @@ const EXAMPLES = {
     'controllers, interceptors, and observers.',
   'lb3-application':
     'An example LoopBack 3 application mounted in a LoopBack 4 project.',
+  'rest-crud':
+    'A simplified version of the Todo example using the default repository' +
+    'and controller without defining classes for them.',
```

Comment:
@raymondfeng do you have any suggestions for how to make this shorter?

(I'll add the missing space after "repository" when I squash the commits)

Reply:
`A simplified version of the Todo example that only requires a model class and a datasource configuration.`?